client/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import "./modal.css";
+
+type ModalProps = {
+  children: JSX.Element;
+};
+
+export const Modal = (props: ModalProps) => {
```

Comment:
can we have show/isVisible prop to Modal, it will require to create the portal if mount/unmount it each time visibility changes